packages/modal-ui/src/lib/components/WalletHome.tsx

```diff
@@ -98,25 +99,126 @@ export const WalletHome: React.FC<WalletHomeProps> = ({
       {route === "GetWallets" && (
         <div className="get-wallet-wrapper">
           {modules.map((module) => {
-            const { iconUrl, name, description } = module.metadata;
+            const { iconUrl, name } = module.metadata;
+            const { type } = module;
+            let typeFullName = "";
+            let haveQR,
+              haveLink = false;
+            switch (type) {
+              case "bridge":
+                typeFullName = "Bridge wallet";
+                break;
+              case "injected":
+                typeFullName = "Injected wallet";
+                haveLink = true;
+                break;
+              case "browser":
+                typeFullName = "Browser Extension";
+                haveQR = true;
+                break;
+              case "hardware":
+                typeFullName = "Hardware wallet";
+                break;
+            }
             return (
-              <div className="single-wallet-get" key={module.id}>
+              <div
+                className="single-wallet-get"
```

Comment:
- The design/css is missing for the Light Mode.
- In `dark mode` the background of these options is  `background: rgba(0, 0, 0, 0.12)` not the same as the modal's background.
- Options are clickable we need to add `cursor: pointer` on hover.


![get-a-wallet](https://user-images.githubusercontent.com/95851345/200114755-2d05037e-d6b5-4a3a-ac3a-ab7817f3146f.PNG)

Reply:
Fixed this.